symfony/framework-bundle/3.3/public/index.php

```diff
@@ -21,7 +21,9 @@
     Debug::enable();
 }
 
-// Request::setTrustedProxies(['0.0.0.0/0'], Request::HEADER_FORWARDED);
+if (($trustedProxies = getenv('TRUSTED_PROXIES')) && ($trustedHeaders = getenv('TRUSTED_HEADERS'))) {
```

Comment:
If you want readability, maybe assign the vars before like this is more readable?

```php
$trustedProxies = getenv('TRUSTED_PROXIES');
$trustedHeaders = getenv('TRUSTED_HEADERS');
if ($trustedProxies && $trustedHeaders) {
    // ...
}
```

I know it's more lines of code, but it's actually the most readable way IMO